Bio/Graphics/GenomeDiagram/_LinearDrawer.py

```diff
@@ -1141,20 +1145,17 @@ def draw_bar_graph(self, graph):
         # taken to specify the number of pixel units resolved along the
         # y-axis
         resolution = max((midval - minval), (maxval - midval))
-        if resolution == 0:
-            resolution = trackheight
+        resolution = resolution if resolution else trackheight
 
         # Create elements for the bar graph based on newdata
         for pos0, pos1, val in data:
             fragment0, x0 = self.canvas_location(pos0)
             fragment1, x1 = self.canvas_location(pos1)
             x0, x1 = self.x0 + x0, self.x0 + x1     # account for margin
             barval = trackheight * (val - midval) / resolution
-            if barval >= 0:  # Different colors for bars that extend above...
-                barcolor = graph.poscolor
-            else:           # ...or below the axis
-                barcolor = graph.negcolor
-
+            # Different colors for bars that extend above...
+            # ...or below the axis
```

Comment:
With a single line of code now, there is no reason to split the comment anymore, e.g.

```python
# Different colors for bars that extend above/below the axis
```